### PR TITLE
fix: Layout issues with FormRecentReposSettings

### PR DESCRIPTION
**Pull request by RussKie**
The existing layout prevented the dialog from being resized to accommodate for longer repository names.
Also fix tab stops and set dialog results.

### CODE REVIEW

**Comment by jbialobr** [comment]
When `Combobox minimum width` is greater than 0, the listboxes should be set to that width. It is done to help visualize how the existing repos names will fit in the set width.

**Comment by RussKie** [comment]
Righto.... thanks
It isn't obvious that this is an intended behavior.

Here's my dropdown:
image

and the configuration dialog:
image

So this is a bug, I suppose.

I will take your note and see how it can be made more obvious and user-friendly at the same time.

**Comment by RussKie** [comment]
image

image

image

**Comment by RussKie** [comment]
outstanding: still need to localise the note under the up/down control

File: GitUI/CommandsDialogs/BrowseDialog/FormRecentReposSettings.Designer.cs [hunk 1]
```diff
@@ -324,69 +351,87 @@ private void InitializeComponent()
...
+            // label2
+            // 
+            tableLayoutPanel1.SetColumnSpan(this.comboMinWidthNote, 2);
+            this.comboMinWidthNote.Dock = System.Windows.Forms.DockStyle.Fill;
+            this.comboMinWidthNote.Location = new System.Drawing.Point(11, 219);
+            this.comboMinWidthNote.Name = "comboMinWidthNote";
+            this.comboMinWidthNote.Padding = new System.Windows.Forms.Padding(12, 0, 0, 0);
+            this.comboMinWidthNote.Size = new System.Drawing.Size(300, 100);
+            this.comboMinWidthNote.TabIndex = 7;
+            this.comboMinWidthNote.Text = "NB: The width of the columns help to visualise how the repository name will be sh" +
```
Comment (jbialobr) [inline GitUI/CommandsDialogs/BrowseDialog/FormRecentReposSettings.Designer.cs]: help -> helps

Comment (RussKie) [inline GitUI/CommandsDialogs/BrowseDialog/FormRecentReposSettings.Designer.cs]: :man_facepalming: 
thanks

**Pull request merged**